tests/exchange/test_liquidation_handler.cairo

```diff
@@ -26,6 +26,7 @@ use satoru::exchange::base_order_handler::BaseOrderHandler::{
 use satoru::event::event_emitter::{IEventEmitterDispatcher};
 
 #[test]
+#[should_panic(expected: ('empty_order',))]
```

Comment:
it should be an empty order and the function name is : given_normal_conditions_when_create_execute_liquidation_then_works